package.json

```diff
@@ -104,5 +104,10 @@
 			"vite": "npm:rolldown-vite@6.3.6"
 		}
 	},
-	"browserslist": ["last 2 versions", "not dead", "not IE 11"]
+	"browserslist": ["last 2 versions", "not dead", "not IE 11"],
+	"dependencies": {
+		"@launchpad-ui/components": "0.9.22",
+		"@launchpad-ui/icons": "0.21.11",
+		"@launchpad-ui/tokens": "0.12.4"
+	}
```

Comment:
Do we need these changes?

Reply:
@traciklopez i forgot about this comment -- this should not have gone in, unless you had a good reason?

Reply:
oh shoot. I thought this was from the changelog command.